posthog/schema.py

```diff
@@ -2505,6 +2506,35 @@ class WebStatsBreakdown(StrEnum):
     FRUSTRATION_METRICS = "FrustrationMetrics"
 
 
+class Metrics(BaseModel):
+    model_config = ConfigDict(
+        extra="forbid",
+    )
+    Bounces: Optional[float] = None
```

Comment:
Let's call this BounceRate, we want to show the avg(is_bounce), rather than sum(is_bounce)